provider/provider_yaml_test.go

```diff
@@ -115,7 +114,6 @@ func TestIamMember(t *testing.T) {
 }
 
 func TestLogSink(t *testing.T) {
-	t.Skipf("TODO[https://github.com/pulumi/pulumi-gcp/issues/2671]: Flaky")
```

Comment:
This is unrelated and does not look correct, as the issues are still open. Can you please remove these changes?